### DISCUSSION TITLE
Translations for fields Title, Description, Tags from "directus_files" collection?

### DISCUSSION THREAD

**Discussion by rynskiy** [directus/directus#15459] [Archive]
Hi,

how can I set translations for default "Image" fields "Title", "Description", "Tags" by collection "directus_files"? 

These fields are closed for edit... This means that if I set the field "translation" by collection "directus_files" with own field "Title", "Description", "Tags" by "directus_files_translations" - default fields cannot be removed...
![default fields translation](https://user-images.githubusercontent.com/54798086/188923658-125e5baf-b170-4319-9aa7-ddcdea0fbf54.PNG)

**Comment by zicklag** [comment]
I don't think you can do that, but you could add your own translated fields to replace the existing ones, granted that's annoying because the existing fields will still pop up in the admin edit page.

**Comment by MarvinXu** [comment]
This is useful for image seo. Are there any other options to provide translations for image descriptions?
